data/publications/index.yaml

```
@@ -19,6 +19,13 @@
     url: https://doi.org/10.26434/chemrxiv-2022-hq0vq
     license: "CC BY 4.0"
     date: 2022-11-04
+   published:
+    doi: https://doi.org/10.1021/acs.jcim.2c01185
+    journal: Journal of Chemical Information and Modeling
+    volume: 62:23
+    pages: 6094–6104
+    year: 2022
+    date: 2022-11-26
   code: openforcefield/qca-dataset-submission/tree/master/submissions/2021-07-28-OpenFF-Industry-Benchmark-Season-1-MM-v1.1
```

```suggestion
  published:
    doi: https://doi.org/10.1021/acs.jcim.2c01185
    journal: Journal of Chemical Information and Modeling
    volume: 62:23
    pages: 6094–6104
    year: 2022
    date: 2022-11-26
  code: openforcefield/qca-dataset-submission/tree/master/submissions/2021-07-28-OpenFF-Industry-Benchmark-Season-1-MM-v1.1
```

This line is incorrectly indented :)